#!/usr/bin/env python
PACKAGE = "course_tutorials"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('hue_low',        int_t,   0, 'Hue Low',            0,   0,  179)
gen.add('hue_high',       int_t,   0, 'Hue High',         179,   0,  179)
gen.add('sat_low',        int_t,   0, 'Sat Low',            0,   0,  255)
gen.add('sat_high',       int_t,   0, 'Sat High',         255,   0,  255)
gen.add('val_low',        int_t,   0, 'Value Low',          0,   0,  255)
gen.add('val_high',       int_t,   0, 'Value High',        255,  0,  255)

exit(gen.generate(PACKAGE, "course_tutorials", "HSVImageProcessDynCfg"))

